crit/crit/__main__.py

```diff
@@ -328,7 +328,7 @@ def explore_rss(opts):
 
             pme = pm['vaddr'] + (pm['nr_pages'] << 12)
             vstr = ''
-            while vmas[vmi]['start'] < pme:
+            while vmi < len(vmas) and vmas[vmi]['start'] < pme:
```

Comment:
It looks like the changes in https://github.com/checkpoint-restore/criu/pull/1491/commits/8be8d145f5255833bac9a758347d4ad0083193a3 somehow got lost. Would you be able to reapply this patch?